### PR TITLE
Upgrade Content API to 5.0

### PR DESCRIPTION
**Pull request by robertberry-zz**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ object Dependencies {
   val awsSdk = "com.amazonaws" % "aws-java-sdk" % "1.8.7"
   val commonsHttpClient = "commons-httpclient" % "commons-httpclient" % "3.1"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
-  val contentApiClient = "com.gu" %% "content-api-client" % "4.1"
+  val contentApiClient = "com.gu" %% "content-api-client" % "5.0"
```
Comment (rich-nguyen) [inline project/Dependencies.scala]: which repo does this live in nowadays? can't find it in typesafe releases.

Comment (robertberry) [inline project/Dependencies.scala]: Content API client? It's on Maven Central

Comment (rich-nguyen) [inline project/Dependencies.scala]: found it, ta

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**